test/e2e-api/main.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2022 The Numaproj Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"net/http"
+)
+
+func main() {
+	if err := http.ListenAndServe(":8378", nil); err != nil {
```

Comment:
Do we need the e2e-api? We neither use e2e-api to send messages nor to verify output. Please see if we can remove the entire `test/e2e-api` folder and `test/manifests` folder.